src/chainparams.cpp

```diff
@@ -121,6 +121,7 @@ class CMainParams : public CChainParams {
         vSeeds.emplace_back("seed.btc.petertodd.org"); // Peter Todd, only supports x1, x5, x9, and xd
         vSeeds.emplace_back("seed.bitcoin.sprovoost.nl"); // Sjors Provoost
         vSeeds.emplace_back("dnsseed.emzy.de"); // Stephan Oeste
+        vSeeds.emplace_back("seed.bitcoin.wiz.biz"); // Jason Maurice
```

Comment:
Should note which x's are supported. I don't recall but you should be able to see it from the zone file. Feel free to update mine while you're at it :)

Reply:
Doesn't dnsseed-rust support all the service bits? In any case, the service bits will change over time, so I'm not sure it makes to hard-code it in the comment, and clearly others feel the same way

Reply:
Right, I just figured match the existing comments, but it doesnt matter.